gradle/gradle-mvn-push.gradle

```diff
@@ -17,23 +17,10 @@
 apply plugin: 'maven'
```

Comment:
i dont remember when why or how i changed this file

Reply:
My guess would be you copied the entire `gradle/` folder from some other project. It definitely needs reverted.